Jenkinsfile

```diff
@@ -88,61 +73,75 @@ pipeline {
+						sh "rm ${previousReleaseCurrentFinalFilename}.gz"
+						
+						// Replace 'release_current' DB with updated 'slice_current' DB.
+				    		def releaseVersion = utils.getReleaseVersion()
+						def sliceCurrentDump = "${env.SLICE_CURRENT_DB}_${releaseVersion}.dump"
+						def databaseDumpFilename = utils.takeDatabaseDumpAndGzip("${env.SLICE_CURRENT_DB}", "update_stable_ids", "after", "${env.RELEASE_SERVER}")
+			            		sh "echo ${databaseDumpFilename}"
+				 		utils.replaceDatabase("${env.RELEASE_CURRENT_DB}", "${databaseDumpFilename}")
 					}
 				}
 			}
 		}
 		// This stage backs up the gk_central and slice_current databases after they have been modified.
-		stage('Post: Backup DBs'){
+		stage('Post: Back up Curator gk_central DB'){
 			steps{
 				script{
-					withCredentials([usernamePassword(credentialsId: 'mySQLUsernamePassword', passwordVariable: 'pass', usernameVariable: 'user')]){
-						withCredentials([usernamePassword(credentialsId: 'mySQLCuratorUsernamePassword', passwordVariable: 'curPass', usernameVariable: 'curUser')]){
-							def slice_current_after_update_stable_ids_dump = "${env.SLICE_CURRENT}_${currentRelease}_after_st_id.dump"
-							def central_after_update_stable_ids_dump = "${env.GK_CENTRAL}_${currentRelease}_after_st_id.dump"
-							sh "mysqldump -u$user -p$pass ${env.SLICE_CURRENT} > $slice_current_after_update_stable_ids_dump"
-							sh "gzip -f $slice_current_after_update_stable_ids_dump"
-							sh "mysqldump -u$curUser -p$curPass -h${env.CURATOR_SERVER} ${env.GK_CENTRAL} > $central_after_update_stable_ids_dump"
-							sh "gzip -f $central_after_update_stable_ids_dump"
-						}
+					withCredentials([usernamePassword(credentialsId: 'mySQLCuratorUsernamePassword', passwordVariable: 'pass', usernameVariable: 'user')]){
+                        			utils.takeDatabaseDumpAndGzip("${env.GK_CENTRAL_DB}", "update_stable_ids", "after", "${env.CURATOR_SERVER}")
 					}
 				}
 			}
 		}
-		// This stage archives all logs and database backups produced by UpdateStableIdentifiers in the Reactome S3 bucket.
-		stage('Archive logs and backups'){
+		// Archives logs and databases on S3, and then everything on the server.
+		stage('Post: Archive Outputs'){
 			steps{
 				script{
-					def s3Path = "${env.S3_RELEASE_DIRECTORY_URL}/${currentRelease}/update_stable_ids"
-					sh "mkdir -p databases/"
-					sh "mv --backup=numbered *_${currentRelease}_*.dump.gz databases/"
-					sh "mv data-release-pipeline/ortho-stable-id-history/logs/* logs/"
-					sh "rm -rf data-release-pipeline*"
-					sh "gzip logs/*"
-					sh "aws s3 --no-progress --recursive cp databases/ $s3Path/databases/"
-					sh "aws s3 --no-progress --recursive cp logs/ $s3Path/logs/"
-					sh "rm -r databases"
-					sh "rm -r logs"
+					def dataFiles = []
+					// Additional log files from post-step QA need to be pulled in
+					def logFiles = ["data-release-pipeline/ortho-stable-id-history/logs/*"]
```

Comment:
I'm not sure I understand why ortho-stable-id-history logs are needed in a step for the update-stable-ids repository.  Is this repo only for manually curated stable ids or is this a relationship to ortho-stable-ids requiring those logs, too

Reply:
Its because the post-step QA exists in that repository. The logger settings have log files saved with the name of the repository they are being executed from